AndresMunoz_DiegoCollazos/PrincipiosSolid/README.md

```diff
@@ -0,0 +1,23 @@
+# Practica: Taller principios SOLID
+
+
+## Integrantes 
+Andrés David Muñoz - Diego Collazos Perafán 
+
+##Proceso en CLI 
+
+1. Limpiar el .jar 
+```
+mvn clean
+```
+2. construir el .jar
+```
+mvn package 
+```
+3. ejecutar el Main
+```
+java -cp target/PrincipiosSOLID-SolucionTarea-1.0-SNAPSHOT.jar com.mycompany
```

Comment:
Actualizar:

```
java -cp target/PrincipiosSOLID-SolucionTarea-1.0-SNAPSHOT.jar co.unicauca.parkinglot.presentacion.ClientMain
```

Reply:
NO funciona:

java -cp target/PrincipiosSOLID-SolucionTarea-1.0-SNAPSHOT.jar co.unicauca.parkinglot.presentacion.ClientMain
Nov 18, 2021 11:12:56 AM co.unicauca.parkinglot.access.VehicleRepository connect
SEVERE: null
java.sql.SQLException: No suitable driver found for jdbc:sqlite::memory:
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:702)
	at java.sql/java.sql.DriverManager.getConnection(DriverManager.java:251)
	at co.unicauca.parkinglot.access.VehicleRepository.connect(VehicleRepository.java:114)
	at co.unicauca.parkinglot.access.VehicleRepository.initDatabase(VehicleRepository.java:97)
	at co.unicauca.parkinglot.access.VehicleRepository.<init>(VehicleRepository.java:86)
	at co.unicauca.parkinglot.access.RepositoryFactory.getRepositoryType(RepositoryFactory.java:28)
	at co.unicauca.parkinglot.presentacion.ClientMain.main(ClientMain.java:28)

Exception in thread "main" java.lang.NullPointerException: Cannot invoke "java.sql.Connection.createStatement()" because "this.conn" is null
	at co.unicauca.parkinglot.access.VehicleRepository.initDatabase(VehicleRepository.java:98)
	at co.unicauca.parkinglot.access.VehicleRepository.<init>(VehicleRepository.java:86)
	at co.unicauca.parkinglot.access.RepositoryFactory.getRepositoryType(RepositoryFactory.java:28)
	at co.unicauca.parkinglot.presentacion.ClientMain.main(ClientMain.java:28)